src/webui/api/torrentscontroller.h

```diff
@@ -28,8 +28,14 @@
 
 #pragma once
 
+#include "base/bittorrent/torrentdescriptor.h"
+#include "base/bittorrent/torrentinfo.h"
+#include "base/global.h"
+#include "base/net/downloadmanager.h"
 #include "apicontroller.h"
 
+inline const QString FILE_SOURCE_PREFIX = u"file:"_s;
```

Comment:
It should be declared in corresponding .cpp file (w/o `inline`).

Reply:
Moved to torrentscontroller.cpp.